packages/data-explorer/src/store/header/actions.ts

```diff
@@ -1,5 +1,28 @@
 import client from '@/lib/client'
-import { DataApiResponse } from '@/types/ApiResponse'
+import { DataApiResponse, isDataApiResponseItem } from '@/types/ApiResponse'
+
+const getBreadCrumbPath = async (parentUrl: string, addBreadcrumb: Function, buildLink: Function): Promise<any> => {
```

Comment:
To keep it consistent: getBreadcrumbPath

Reply:
done